Scriptorium-CSC309/web
# Hide blogposts for other users (but not author)

**Summary**

Admins can hide blog posts for all users except the author, who can still see the authored post.

**Testing**

Precondition: database has one blogpost
1. Fetch all blog posts via API (un-authorized, should see precondition post)
2. Login (as admin) via API
3. Create a blog post
4. Hide new post via hide API
5. Use same endpoint as step 1 to fetch (should see precondition post + new post)
6. Login using another user, fetch posts again, should see only precondition post but not new post.

Issues to be closed by this PR: 
- https://github.com/Scriptorium-CSC309/web/issues/49
